### PR TITLE
Cleanup README, add logging, fix failure with no concurrency setting

### PR DESCRIPTION
**Pull request by dagrayvid**

### CODE REVIEW

File: s3storage.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def retrieve_object_body(self, key):
                 Bucket=self.bucket, Key=key)
             content = res.get('Body').read().decode('utf-8')
         except botocore.exceptions.ClientError as err:
-            logging.error(err)
+            logging.debug(err)
```
Comment (kpouget) [inline s3storage.py:129]: these errors are basically swallowed, (invisible if the debug logs are not printed) 🤔 is it part of a try loop?

File: load_test.py [hunk 2]
```diff
@@ -463,8 +467,12 @@ def run_multiplexed(config):
     threads = config.get_threads()
     load_gen_config = config.get_load_gen_config()
     concurrency_per_thread = load_gen_config.get("concurrency")
-    total_conc = threads * concurrency_per_thread
-    print(f"Parallel experiment with {threads} threads and  {concurrency_per_thread} concurrency, for a total concurrency of {total_conc}")
+    if concurrency_per_thread is not None:
+        total_conc = threads * concurrency_per_thread
+        logging.info(f"Parallel experiment with {threads} threads and  {concurrency_per_thread} concurrency, for a total concurrency of {total_conc}")
+    else:
+        print(f"Parallel experiment with {threads} threads and no concurrency setting (ghz default: 50). Use RPS at your own risk")
```
Comment (kpouget) [inline load_test.py:474]: shouldn't this be a `logging.info`?

**Review by kpouget — APPROVED** [review]
thanks David, looks good

File: s3storage.py (hunk 1 above)
Comment (dagrayvid) [reply s3storage.py]: Thanks! Fixed

File: load_test.py (hunk 2 above)
Comment (dagrayvid) [reply load_test.py]: Yes, fixed!

**Pull request merged**